docs/run/runtime-options-overview.mdx

```diff
@@ -0,0 +1,559 @@
+| dynamical_decoupling |                          |                         |                                                                                           |   `FieldInfo(annotation=Union[DynamicalDecouplingOptions, Dict], required=False, default_factory=DynamicalDecouplingOptions)`                         |
+|                      | enable                   |                         | `True`/`False`                                                             |    `False`           |
+|                      | sequence_type            |                         | `'XX'`/`'XpXm'`/`'XY4'`                                                                         | `'XX'`                      |
+|                      | extra_slack_distribution |                         | `'middle'`/`'edges'`                                                                          | `'middle'`                  |
+|                      | scheduling_method        |                         | `'asap'`/`'alap'`                                                                            | `'alap'`                    |
+| environment          |                          |                         |                                                                                                         |                                  |
+|                      | log_level                |                         |`DEBUG`/`INFO`/`WARNING`/`ERROR`/`CRITICAL`                                                              |`WARNING`                          |
+|                      | callback                 |                         |Callable function that receives Job ID and Job result.                                                   |`None`                           |
+|                      | job_tags                 |                         |List of tags                                                                                             |`None`                           |
+| execution            |                          |                         |                                                                                                         |                           |
+|                      | init_qubits              |                         | `True`/`False`                                                                         | `True`                    |
+|                      | rep_delay                |                         | Value in the range supplied by `backend.rep_delay_range`.                                | Given by `backend.default_rep_delay`.|
+| max_execution_time   |                          |                         | integer number of seconds in the range (0, 5000]                                                                   | `10800` (3 hours)                         |
```

Comment:
Just out of curiosity, where did you get the `5000`? The valid range is actually [1, 10800]. Side note, is there a standard way to specify a range? Do we use `()` or `[]` or something else?